### PR TITLE
Refactor navbar burger click action

### PR DESCRIPTION
**Pull request by mathewTH**
Use react element onClick and NavBar react component state to
make burger click event handling simpler and more readable

I was also going to extract a navBurger component (see my other branch) haven't got that working yet.
while I'm at it, perhaps consider using https://github.com/couds/react-bulma-components if you haven't already

### CODE REVIEW

**Comment by manukautrampingclub** [comment]
Burger not changing visual state on open

**Comment by mathewTH** [comment]
😞 lost a change somewhere in the git staging (with all the separate changes i was doing). Fixed and tested with a local deploy

File: src/components/navbar.js [hunk 1]
```diff
@@ -40,38 +42,14 @@ const Navbar = () => {
...
-	})
-
 	return (
 		<nav className="navbar is-light is-fixed-top is-spaced container" role="navigation" aria-label="main navigation">
 			<div className="navbar-brand">
-				<a role="button" className="navbar-burger burger is-marginless" aria-label="menu" aria-expanded="false" data-target="navbarMenu">
+				<a role="button" className={`navbar-burger burger is-marginless ${ isBurgerActive ? 'is-active' : ''}`} aria-label="menu" aria-expanded="false"
+					onClick={() => {
+						setIsBurgerActive(!isBurgerActive);
+						document.getElementById("navbarMenu").classList.toggle('is-active');
```
Comment (mathewTH) [inline src/components/navbar.js:51]: This line isn't really the react way.
I was tempted to do the class thing directly on the navBarMenu (same as on the a tag) but this links the logic together nicely.

Comment (manukautrampingclub) [reply src/components/navbar.js:51]: Yeah I see what you're saying. I think it is fine for now.

**Pull request merged**